SGP.NET/SGP.NET.csproj

```diff
@@ -25,6 +25,10 @@ Remove unused time parameter in Coordinate's GetFootprint,
 Made constants constant</PackageReleaseNotes>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.4|AnyCPU'">
+    <DocumentationFile>C:\Users\Administrator\source\repos\SGP.NET\SGP.NET\SGP.NET.xml</DocumentationFile>
+  </PropertyGroup>
+
```

Comment:
Heads up, this hard-coded absolute path will make the build fail for most people (as it did for me).

Reply:
Strange that it auto-generated an absolute path. I'll fix it before I merge.